services/clusters-keeper/src/simcore_service_clusters_keeper/core/settings.py

```diff
@@ -252,6 +252,14 @@ class PrimaryEC2InstancesSettings(BaseCustomSettings):
         ),
     ] = "172.20.0.0/14"  # nosec
 
+    PRIMARY_EC2_INSTANCES_RABBIT: Annotated[
+        RabbitSettings | None,
```

Comment:
"I believe here rabbit can indeed be truly optional, as the service would still function without it